clusterloader2/testing/network/config.yaml

```diff
@@ -0,0 +1,40 @@
+  {{$Replicas := .Replicas}}
+  {{$Ratio := .Ratio}}
+  {{$Protocol := .Protocol}}
+name: network
+automanagedNamespaces: 1
+tuningSets:
+- name: Sequence
+  parallelismLimitedLoad:
+    parallelismLimit: 1
+steps:
+- name: start network metric measurement
+  measurements:
+    - Identifier: NetworkPerformanceMetrics
+      Method: NetworkPerformanceMetrics
+      Params:
+        action: start 
+        podReplicas: {{$Replicas}}
+        duration: 10
```

Comment:
what is duration? isn't it of "Duration type"? (i.e. shouldn't it have units?"

Reply:
yes i have modified it to duration type here , as well as in code. But while sending it to worker pod via CR , i convert it into seconds.